.editorconfig

```diff
@@ -7,15 +7,5 @@ charset = utf-8
 indent_style = tab
 trim_trailing_whitespace = true
 
-[{*.yml,*.yaml}]
+[*.{yml,yaml}]
 indent_style = space
-indent_size = 2
```

Comment:
This will for example cause all `lang/*.yaml` files to be reformatted as the default seems to be 4 spaces. They are indented with 2 spaces currently. 

Or is the default of 4 a local editor setting? 

In any case I suggest keeping the `indent_size = 2` here.

Reply:
Formatting happens (or is at least intended to) via `prettier` which defaults to an indent size of 2 anyways. I did a quick test with `editorconfig-check` and it doesn't seem to complain about the `lang/*.yaml` files. How did you check it on your end? Do you have any other configs in place which could interfere?

Reply:
I see! I did indeed not rely on prettier in my editor. Sorry for the false alarm then!

Reply:
Of course, the goal is that it works with all editors (which do respect prettier / editorconfig). With the current configuration, are those files mistakenly reformatted with 4 spaces in your case?

Reply:
I turned on prettier in WebStorm, no idea why it wasn't picked up by default. Now it works like a charm and no reformatting happens.

Reply:
Okay, great! Thanks for your confirmation!